### PR TITLE
[JW8-11180] Set track mode to hidden for iOS fullscreen

### PR DESCRIPTION
**Pull request by pajong**
### This PR will...
- Set `track.mode` to `hidden` for iOS native rendered captions

### Why is this Pull Request needed?
- For fullscreen iOS, setting `track.mode` to `disabled` does not hide the captions already displayed, resulting captions to be stuck while playing ads

#### Addresses Issue(s):

JW8-11180

### Checklist
- [ ] Jenkins builds and unit tests are passing
- [ ] I have reviewed the automated results

### CODE REVIEW

File: src/js/providers/tracks-mixin.ts [hunk 1]
```diff
@@ -214,7 +214,7 @@ const Tracks: TracksMixin = {
             // FF does not remove the active cue from the dom when the track is hidden, so we must disable it
```
Comment (robwalch) [inline src/js/providers/tracks-mixin.ts:214]: This comment looks like it needs to be updated.

File: src/js/providers/tracks-mixin.ts [hunk 2]
```diff
@@ -211,12 +211,13 @@ const Tracks: TracksMixin = {
     disableTextTrack(): void {
         const track = this.getCurrentTextTrack();
         if (track) {
-            // FF does not remove the active cue from the dom when the track is hidden, so we must disable it
-            track.mode = 'disabled';
+            // IOS native captions does not remove the active cue from the dom when the track is disabled, so we must hide it
             const trackId = track._id;
-            if (trackId && trackId.indexOf('nativecaptions') === 0) {
+            if ((trackId && trackId.indexOf('nativecaptions') === 0) || (this.renderNatively && OS.iOS)) {
                 track.mode = 'hidden';
```
Comment (zetagame) [inline src/js/providers/tracks-mixin.ts:217]: what does this line do if we set to disabled right after either way?

Comment (pajong) [reply src/js/providers/tracks-mixin.ts:217]: As soon as we set to `hidden`, the text track is hidden on iOS. We want to set to disable as per Rob's comment above:

 `Could we hide the track and then disable it? Leaving a track hidden requires that the track continue to be processed (loading and parsing VTT files... etc.). The only subtitles track that should not be disabled is the one being shown.`

**Comment by pajong** [comment]
test this please

**Pull request merged**